src/GraphQl/Resolver/Factory/ItemMutationResolverFactory.php

```diff
@@ -79,6 +79,10 @@ public function __invoke(string $resourceClass = null, string $rootClass = null,
             $normalizationContext = $resourceMetadata->getGraphqlAttribute($operationName ?? '', 'normalization_context', [], true);
             $normalizationContext['attributes'] = $this->fieldsToAttributes($info);
 
+            if ('delete' !== $operationName && isset($normalizationContext['attributes'][lcfirst($resourceMetadata->getShortName())])) {
```

Comment:
you should store the result of `lcfirst($resourceMetadata->getShortName())` in a variable instead of computing it 3 times.